setup.py

```diff
@@ -1,6 +0,0 @@
-# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
Heey! Good changes.
Just one thing, why are we removing this file?

Reply:
Hi @Nil-Andreu , if my understanding is correct the new way of "packaging" Python project is to use the pyproject.toml instead of the setup.* files. Maybe I am wrong?

Reply:
Ah yes true, just researched it and seems a new update! Thanks and sorry for the inconvenience!

Reply:
It is not a problem at all. I have learned about pyproject.toml recently too. I wasn't aware.